include/zephyr/bluetooth/mesh/access.h

```diff
@@ -108,9 +108,6 @@ extern "C" {
 
 /** Abstraction that describes a Mesh Element */
 struct bt_mesh_elem {
-	/** Unicast Address. Set at runtime during provisioning. */
```

Comment:
What is the reason to change public API that is matured and used in many products that are built above Zephyr's Mesh? 
That should be the meaningful reason to start public API changing with the deprecation procedure described in Zephyr's procedure documentation.

https://docs.zephyrproject.org/3.2.0/develop/api/api_lifecycle.html

Reply:
> What is the reason to change public API that is matured and used in many products that are built above Zephyr's Mesh?

I think the definition of the element should be set to const from the beginning, which is my idea.